logging.go

```diff
@@ -29,15 +33,18 @@ var logLevelFormatters = map[string]string{
 	DevelLogLevel:  "%+v",
 }
 
+var LogWriters = []string{CsvPg, JsonPg, TextPg, TextWalg}
+var logWriter = TextWalg
+
 func setupLoggers() {
-	if logLevel == NormalLogLevel {
-		DebugLogger = NewErrorLogger(ioutil.Discard, "DEBUG: ")
-	} else if logLevel == ErrorLogLevel {
-		DebugLogger = NewErrorLogger(ioutil.Discard, "DEBUG: ")
-		InfoLogger = NewErrorLogger(ioutil.Discard, "INFO: ")
-		WarningLogger = NewErrorLogger(ioutil.Discard, "WARNING: ")
-	} else {
-		DebugLogger = NewErrorLogger(os.Stdout, "DEBUG: ")
+	if logWriter == TextWalg {
+		setupWalgLoggers()
+	} else if logWriter == JsonPg {
+		setupJsonPgLoggers()
```

Comment:
why need to use separate functions? Can we just setup the writer and field values and then send it to setupLoggers()?